pkg/tagger/local/tagstore.go

```diff
@@ -69,11 +72,39 @@ func newTagStore() *tagStore {
 	return &tagStore{
 		telemetry:  make(map[string]map[string]float64),
 		store:      make(map[string]*entityTags),
+		infoIn:     make(chan []*collectors.TagInfo, 5),
```

Comment:
We should increase this to something like `20` or `50` to make sure the collectors do not wait in case this goroutine is not scheduled for some time.